core/network/helpers/scale_message_read_writer.cpp

```diff
@@ -12,6 +12,6 @@ namespace kagome::network {
 
   ScaleMessageReadWriter::ScaleMessageReadWriter(
       const std::shared_ptr<libp2p::basic::ReadWriter> &read_writer)
-      : read_writer_{
-          std::make_shared<libp2p::basic::MessageReadWriter>(read_writer)} {}
+      : read_writer_{std::make_shared<libp2p::basic::MessageReadWriterUvarint>(
```

Comment:
Not cleat why not pass MessageReadWriterUvarint in the first place.

Reply:
we need to wrap it with message read writer which is implementation detail. No need to pass it as argument